docs/api/browser-window.md

```diff
@@ -547,6 +547,32 @@ Calling `window.print()` in web page is equivalent to call
 doesn't need print feature, you can safely remove `pdf.dll` in saving binary
 size.
 
+### BrowserWindow.printToPDF([options], callback)
+
+* `options` Object
+  * `marginsType` Integer - Specify the type of margins to use
+    * 0 - default
+    * 1 - none
+    * 2 - minimum
+  * `shouldPrintBackgrounds` Boolean - Whether to print CSS backgrounds.
```

Comment:
We should use `printBackground` as option name to keep consistent with `BrowserWindow.print`.